src/lib/tabs/tab-group.ts

```diff
@@ -134,9 +134,16 @@ export class MdTabGroup extends _MdTabGroupMixinBase implements AfterContentInit
   /** Event emitted when focus has changed within a tab group. */
   @Output() focusChange: EventEmitter<MdTabChangeEvent> = new EventEmitter<MdTabChangeEvent>();
 
-  /** Event emitted when the tab selection has changed. */
+  /**
+   * Event emitted when the tab selection has changed.
+   * @deprecated Use `selectedTabChange` instead.
+   */
   @Output() selectChange: EventEmitter<MdTabChangeEvent> = new EventEmitter<MdTabChangeEvent>(true);
 
+  /** Event emitted when the tab selection has changed. */
+  @Output() selectedTabChange: EventEmitter<MdTabChangeEvent> =
```

Comment:
Rather than duplicating the creation of the EventEmitter and then having to call both, why not just have `selectedTabChange` get the `selectChange` value currently.

```ts

/**
 * Event emitted when the tab selection has changed.
 * @deprecated Use `selectedTabChange` instead.
 */
@Output() get selectChange(): EventEmitter<MdTabChangeEvent> {
  return this.selectedTabChange;
}

/** Event emitted when the tab selection has changed. */
@Output() get selectedTabChange(): EventEmitter<MdTabChangeEvent> = 
  new EventEmitter<MdTabChangeEvent>(true);
```

Doing it this way, allows for users to move to the new output, and ultimately allows for the removal of the `@Output` with minimal change to the code in the future as no change is needed below in the file.

Reply:
Thanks for the suggestion. Updated!